coord/src/rmr/conn.c

```diff
@@ -32,7 +32,7 @@ static int MRConn_SendAuth(MRConn *conn);
 #define RSCONN_REAUTH_TIMEOUT 1000
 
 #define CONN_LOG(conn, fmt, ...)                                                \
-  fprintf(stderr, "[%p %s:%d %s]" fmt "\n", conn, conn->ep.host, conn->ep.port, \
+  RedisModule_Log(NULL, "notice", "[%p %s:%d %s]" fmt "\n", conn, conn->ep.host, conn->ep.port, \
```

Comment:
I am not sure if we should do this change on a patch version, WDYT?

Reply:
I don't think it is bad to add logs on a patch release. We can make them `verbose` or `debug` if we prefer to ignore them by default

Reply:
The thing is that the code uses the same `CONN_LOG` macro for all logs, but actually some of them should probably be debug while others probably should be errors. Maybe we should make this change on another PR and make it properly...